src/Uno.UI.Runtime.Skia.Gtk/UI/Xaml/Controls/MultilineTextBoxView.cs

```diff
@@ -0,0 +1,59 @@
+﻿#nullable enable
+
+using System;
+using Gtk;
+using Uno.Disposables;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.UI.Runtime.Skia.UI.Xaml.Controls;
+
+internal class MultilineTextBoxView : GtkTextBoxView
+{
+	private readonly ScrolledWindow _scrolledWindow = new();
+	private readonly TextView _textView = new();
+
+	public MultilineTextBoxView()
+	{
+		_scrolledWindow.Add(_textView);
+	}
+
+	protected override Widget RootWidget => _scrolledWindow;
+
+	protected override Widget InputWidget => _textView;
+
+	public override string Text
+	{
+		get => _textView.Buffer.Text;
+		set => _textView.Buffer.Text = value;
+	}
+
+	public override bool IsCompatible(TextBox textBox) => textBox.AcceptsReturn && textBox is not PasswordBox;
+
+	public override (int start, int end) GetSelectionBounds() => _textView.Buffer.GetSelectionBounds(out var start, out var end) ? (start.Offset, end.Offset) : (0, 0); // TODO: Confirm this implementation is correct.
+
+	public override void SetSelectionBounds(int start, int end)
+	{
+		var startIterator = _textView.Buffer.GetIterAtOffset(start);
+		var endIterator = _textView.Buffer.GetIterAtOffset(end);
+		_textView.Buffer.SelectRange(startIterator, endIterator);
+	}
+
+	public override void UpdateProperties(TextBox textBox)
+	{
+		base.UpdateProperties(textBox);
+
+		_textView.Editable = !textBox.IsReadOnly;
```

Comment:
Should this be `!textBox.IsReadOnly && textBox.IsTabStop`?

Reply:
Technically it should not matter, as in case of `IsTabStop` being false it should not even be possible to focus that `TextBox` and trigger the native view, but will add it to be sure 👍

Reply:
@MartinZikmund IIRC, the `IsTabStop` was needed for Android at least. I'm fine with not adding `IsTabStop` as long as you verify the behavior is correct (with a test *if possible*)